### DISCUSSION TITLE
[VSCode 빌드 실패] cannot open source file "rclcpp/rclcpp.hpp" 오류로 인한 빌드 실패

### DISCUSSION THREAD

**Discussion by KimHyeJun1** [Road-Balance/gcamp_ros2_basic#61] [Q&A] [answered]
프로그래밍 공부 중에 basic_topic_pkg 빌드를 하려 하는데 아래 그림과 같이 빌드가 3분 동안 빌드를 합니다. 원래는 금방 끝났었는데요
![image](https://user-images.githubusercontent.com/109134044/179897093-649982c0-f12a-4131-9e1a-701c78e83ca1.png)

vscode에 들어가 보니까 문제가 있었습니다.
![image](https://user-images.githubusercontent.com/109134044/179897179-0e5cf91b-c171-4f78-93c0-5cdfb021dd4d.png)

위 문제 때문에 빌드가 오래 걸리는 것 같은데 소스를 열 수 없다고 합니다 어떤 이유에서 그런 건가요? 그리고 해결은 Please update your include path라고 하는데 어떻게 해야하나요?

**Comment by kimsooyoung** [comment, marked as answer]
윈도우즈에서 rclcpp을 인식하지 못하는 것은 정상입니다. 

제 경험상 해당 상황은 새로운 터미널을 실행하거나, 재부팅 시 해결되었습니다. 

간혹 발생하지만 빌드 자체가 실패하는 것은 아니라 감수하고 사용해야 할 것 같습니다.
